### PR TITLE
Add inputbox for android

### PR DESCRIPTION
**Pull request by xiushudongfang**
For inputting  IP and NickName ,or something else in the future on Android

### CODE REVIEW

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -472,6 +472,8 @@ void GameSettingsScreen::CreateViews() {
 
 #ifdef _WIN32
 	networkingSettings->Add(new PopupTextInputChoice(&g_Config.proAdhocServer, n->T("Change proAdhocServer Address"), "", 255, screenManager()));
+#elif defined(ANDROID)
+	networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.proAdhocServer, sy->T("Change proAdhocServer Address"), nullptr))->OnClick.Handle(this, &GameSettingsScreen::OnChangeproAdhocServerAddress);
```
Comment (unknownbrackets) [inline UI/GameSettingsScreen.cpp]: Just leave the existing one?

```c++
networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.proAdhocServer, n->T("Change proAdhocServer Address"), nullptr))->OnClick.Handle(this, &GameSettingsScreen::OnChangeproAdhocServerAddress);
```

We want to use `n->` and the lines are the same.

-[Unknown]

File: UI/GameSettingsScreen.cpp [hunk 2]
```diff
@@ -912,6 +916,8 @@ UI::EventReturn GameSettingsScreen::OnChangeNickname(UI::EventParams &e) {
 	if (System_InputBoxGetString("Enter a new PSP nickname", g_Config.sNickName.c_str(), name, name_len)) {
 		g_Config.sNickName = name;
 	}
+#elif defined(ANDROID)
+	System_SendMessage("inputbox", ("nickname :" + g_Config.sNickName).c_str());
```
Comment (unknownbrackets) [inline UI/GameSettingsScreen.cpp]: Do these need to be translated?  I think the title is shown to the user.

Also, why the trailing space before `:`?

-[Unknown]

File: UI/GameSettingsScreen.cpp (hunk 1 above)
Comment (xiushudongfang) [inline UI/GameSettingsScreen.cpp]: Sorry ,I made a mistake.

File: UI/GameSettingsScreen.cpp (hunk 2 above)
Comment (xiushudongfang) [inline UI/GameSettingsScreen.cpp]: The title is in java code.

**Pull request merged**